<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbSend">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Message&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <zorder>textEdit_input</zorder>
   <zorder>label</zorder>
   <zorder>textEdit_log</zorder>
   <zorder>pbSend</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
